WordPress/Classes/ViewRelated/Comments/CommentViewController.m

```diff
@@ -641,8 +641,20 @@ - (void)sendReplyWithNewContent:(NSString *)content
     };
     
     void (^failureBlock)(NSError *error) = ^void(NSError *error) {
-        
-        NSString *message = NSLocalizedString(@"There has been an unexpected error while sending your reply", nil);
+        NSUInteger lastIndex = content.length == 0 ? 0 : content.length - 1;
+        NSUInteger composedCharacterIndex = NSMaxRange([content rangeOfComposedCharacterSequenceAtIndex:MIN(lastIndex, 140)]);
+        // 140 is a somewhat arbitraily chosen number (old tweet length) — should be enough to let people know what
+        // comment failed to show, but not too long to display.
+
+        NSString *replyExcerpt = [content substringToIndex:composedCharacterIndex];
+
+        NSString *message = [NSString stringWithFormat:NSLocalizedString(@"There has been an unexpected error while sending your reply: \n\"%@\"", nil), replyExcerpt];
```

Comment:
I believe that `NSLocalizedString(@"There has been an unexpected error while sending your reply: \n\"%@\""` will count as a new string to be localized because of the added `: \n\"%@\"` symbols. Maybe it would be good to keep the localized string as it was before to keep using the old translations?

Its an extra step in the three places where it was changed but it might be worth it :)

Reply:
Sounds good. Thank you for the explanation!